app/services/create_requested_account.rb

```diff
@@ -26,6 +26,19 @@ def initialize(requested_account, creator)
     process_request(@creator, @creation_reason)
   end
 
+  def set_result_description
+    if @result[:failure]
+      @result[:result_description] = @result[:failure].sub('response: {}', '')
```

Comment:
This is still a bit confusing, because you're writing to `@result[:result_description]` twice, first with an intermediate string and then with the final i18n message.

Better to use a temporary variable here. Why are we substituting the URL? The name of the temporary variable should give some indication of the purpose of this step.

Reply:
Done!
The url and empty response substrings are removed from the received message from the server so as to display them in user friendly format. Otherwise they would look like the failure string in the Before image of the pull request (Image in the starting).